decidim-comments/lib/decidim/comments/commentable.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require "active_support/concern"
+
+module Decidim
+  module Comments
+    # Shared behaviour for commentable models.
+    module Commentable
+      extend ActiveSupport::Concern
+
+      included do
+        include Decidim::Authorable
+
+        has_many :comments, as: :commentable, foreign_key: "decidim_commentable_id", foreign_type: "decidim_commentable_type", class_name: "Decidim::Comments::Comment"
+
+        def commentable?
```

Comment:
We should add docs indicating what all these defaults mean and what effect they have on comments.